### DISCUSSION TITLE
Do we have an ETA on when dev getting merged to main?

### DISCUSSION THREAD

**Discussion by ye7iaserag** [d8ahazard/sd_dreambooth_extension#944] [Q&A]
I heard that there are many fixes on dev that can be useful, not trying to push or anything but just trying to know if I should just wait for an official merge or if I should just try dev branch.
Is there an announced date? or a plan the maintainers follow before merging?

**Comment by ye7iaserag** [comment]
Just noticed there is a PR https://github.com/d8ahazard/sd_dreambooth_extension/pull/939

**Comment by 78Alpha** [comment]
That pull you mentioned was merged 11 days ago
